runtime/src/accounts.rs

```diff
@@ -690,6 +690,15 @@ impl Accounts {
     }
 }
 
+pub fn create_accounts(accounts: &Accounts, pubkeys: &mut Vec<Pubkey>, num: usize) {
```

Comment:
Fwiw, I think you could import this function into your bench with `use crate::accounts::tests::create_accounts` and not have to move it. It would still need to be `pub`, though

Reply:
I was just about to comment about this change :slightly_smiling_face: 

The problem I ran into is that the `test` build profile isn't in affect when running the bench, so the symbol doesn't exist.  I couldn't find any reference to a similar attribute to enable for `bench`

Reply:
Ahh, 👌